File: Calculator.cs
using System;
using System.Windows.Forms;

namespace CalculatorApp
{
    public partial class Calculator : Form
    {
        public Calculator()
        {
            InitializeComponent();
        }

        private void EvaluateExpression()
        {
            try
            {
                double result = Eval(entryTextBox.Text);
                resultLabel.Text = "Result: " + result;
            }
            catch
            {
                resultLabel.Text = "Invalid expression";
            }
        }

        private double Eval(string expression)
        {
            return Convert.ToDouble(new System.Data.DataTable().Compute(expression, ""));
        }

        private void evaluateButton_Click(object sender, EventArgs e)
        {
            EvaluateExpression();
        }
    }
}

File: calculator.py
import tkinter as tk

def evaluate_expression():
    try:
        result = eval(entry.get())
        result_label.config(text="Result: " + str(result))
    except:
        result_label.config(text="Invalid expression")

# Create the main window
window = tk.Tk()
window.title("Calculator")

# Create the UI elements
entry = tk.Entry(window, width=30)
entry.pack(pady=10)

evaluate_button = tk.Button(window, text="Evaluate", command=evaluate_expression)
evaluate_button.pack()

result_label = tk.Label(window, text="Result: ")
result_label.pack(pady=10)

# Start the main loop
window.mainloop()
